reg/drone/srv/air_data_computer/_.0.1.uavcan

```diff
@@ -0,0 +1,48 @@
+# A generic air data computer (ADC) service.
+#
+# An air data computer service is generally a non-interactive publish-only service: when activated,
+# it keeps publishing its measurements at a fixed rate over several subjects until deactivated.
+# The activation/deactivation, if supported, is managed via the standard arming command service.
+# The data update rates and other parameters are controlled via the Register API using implementation-defined
+# register names.
+#
+# An ADC whose arming state is SLEEP is allowed to cease all network activity.
+#
+# An ADC whose arming state is DISARMED is recommended to behave as if its state was ARMED;
+# implementations are recommended to leverage this state to perform any necessary maintenance activities
+# such as calibration, self-heating, etc. The ADC should not report its own status as ARMED until said
+# maintenance activities have been completed. Therefore, the high-level controler (e.g., the flight
+# management unit) may delay the take-off until the ADC is ready.
+#
+# An ADC whose arming state is ARMED is required to comply with the requirements set out below.
+#
+# An armed ADC should publish the following data subjects synchronously at the same configurable rate
+# which should not be lower than the specified limit. The measurements should be low-pass filtered
+# to avoid frequency aliasing effects.
+#
+#   PUBLISHED SUBJECT       TYPE                                                NOTE
+#   calibrated airspeed     reg.drone.phy.kinematics.trans.Velocity1CovTs
+#   true airspeed           reg.drone.phy.kinematics.trans.Velocity1CovTs       optional
+#   pressure altitude data  reg.drone.phy.kinematics.trans.LinearCovTs          assume ISA; NED frame (+down, -up)
+#   static air data         reg.drone.phy.thermodynamics.PressureTempCovTs      pressure and OAT
+#
+# Observe that there is no subject for indicated airspeed. This is because per the principles of service-oriented
+# design, the ADC is responsible for applying the necessary transformations on its data to render it ready for
+# consumption by clients. If desired, indicated airspeed may be reported over an implementation-defined subject.
+#
+# In addition to the above, an armed ADC should publish the following service subjects at an implementation-defined
+# rate that should not be lower than the specified limit:
+#
+#   PUBLISHED SUBJECT       TYPE
+#   heartbeat               reg.drone.srv.common.Heartbeat
+#
+# Despite being a non-interactive service, an ADC is recommended to subscribe to the arming command subject.
+# This recommendation may be omitted if the ADC does not support arming state selection, in which case it should
+# always report itself as being ARMED.
```

Comment:
Again, let's rethink this concept of arm and replace it with activation for sensors.

Reply:
Moved to the forum https://forum.uavcan.org/t/service-readiness-level/967